src/server/components/lists/helpers.ts

```diff
@@ -220,6 +220,16 @@ export function createFormRunnerEditListItemLink(token: string): string {
   return `${protocol}://${FORM_RUNNER_PUBLIC_URL}${FORM_RUNNER_INITIALISE_SESSION_ROUTE}/${token}`;
 }
 
+function restoreSpecialCharacter(specialCharacter: string, country: string, countryName: string): string {
+  const index = country.indexOf(specialCharacter);
```

Comment:
I feel like [string.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) might be a better fit here. Although indexOf works as well.